src/styles/page-template/index.md.njk

```diff
@@ -117,6 +117,11 @@ Variables can be set with:
       <td class="govuk-table__cell">Add a class to the <code>&lt;body&gt;</code> element.</td>
     </tr>
 
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell">bodyAttributes</td>
+      <td class="govuk-table__cell">Add an attribute to the <code>&lt;body&gt;</code> element.</td>
```

Comment:
Couple of changes suggested, because:
- "an attribute" didn't seem accurate because you can add more than one
- I think it's worth clarifying how to add each attribute, because `bodyAttribute` uses key-value pairs rather than a single value.

```suggestion
      <td class="govuk-table__cell">Add attributes to the <code>&lt;body&gt;</code> element. Add each attribute and its value as a key value pair in the `bodyAttribute` object.</td>
```